en/includes/references/conditional-auth/api-reference.md

```diff
@@ -1009,13 +1015,15 @@ The HTTP POST function enables sending HTTP POST requests to specified endpoints
             "Accept": "application/json"
         }, authConfig, {
             onSuccess: function(context, data) {
-                Log.info('httpPost call succeeded');
-                context.selectedAcr = data.status;
```

Comment:
Refer to my comment on the same functions in httpGet

Reply:
Fixed with https://github.com/wso2/docs-is/pull/4773/commits/9acc22f1d66873e2c67cb71939862187674bece9